### PR TITLE
README.md: little stylistic fixes

### PR DESCRIPTION
**Pull request by seletskiy**
Don't see reason for calling it `madness`.

Also, I think it's bloody good idea to add comparison with top vendoring tools that are already exist.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,135 +2,137 @@
 
 ![madness](https://cloud.githubusercontent.com/assets/8445924/10410421/ccca8b24-6f30-11e5-9952-9e5be5c4d792.png)
 
-Manul is the madness dependencies vendoring utility for Golang programs.
+Manul is the dependency vendoring utility for Golang programs.
```
Comment (kovetskiy) [inline README.md:5]: please return back `madness`

File: README.md [hunk 2]
```diff
@@ -2,135 +2,137 @@
...
 
-## Why yet another utility
+## What the reason for yet another utility?
```
Comment (kovetskiy) [inline README.md]: please do not use `?` in headers

File: README.md [hunk 3]
```diff
@@ -2,135 +2,137 @@
...
-    * You will not be able to find anything using GitHub Search, because you
-        will get false-positive results.
-    * After manually updating libraries you will get weird commits with
-        fantastic line adding/removing counters.
-    * You will not be able to say what version of dependency you use, or you
-        will have additional ambigious file with associative of vendor and
-        commit.
+- Some copy source code of dependencies into the vendor directory:
+    * It will be nearly impossible to find anything ussing GitHub Search or
+        standard search tools like `grep`, because you will get many
```
Comment (kovetskiy) [inline README.md]: please remove notes about grep

File: README.md [hunk 4]
```diff
@@ -2,135 +2,137 @@
...
 
 - Various architecture problems:
-    * Can't update all or specific vendored dependencies
-    * Can't rollback vendored dependencies changes
-    * Can't remove unused vendored dependencies
-    * Can't keep version of vendored dependency
+    * Impossible to update all or specific vendored dependencies;
+    * Impossible to rollback vendored dependencies to specific version;
+    * Impossible to remove unused vendored dependencies;
+    * Impossible to lock version of vendored dependency;
```
Comment (kovetskiy) [inline README.md]: should be `.` insead of `;` here

File: README.md [hunk 5]
```diff
@@ -2,135 +2,137 @@
...
+After installation we can have a look for vendored and non-vendored
+dependencies by using flag `-Q`. After previous step we should see git commits
+along with two already vendored dependencies (`zhash` and `blackfriday`):
 
 ![query after install](https://cloud.githubusercontent.com/assets/8445924/10285719/a39282e4-6b79-11e5-8877-7fba19e0d8c0.gif)
 
 Let's install submodules for remaining dependencies, go the limit! Just run
 **manul** with flag `-I` without specifying any dependencies, manul will
-install all detected dependencies with skipping already-vendored.
+install all detected dependencies with skipping already vendored:
```
Comment (kovetskiy) [inline README.md:86]: Why not `already-vendored`? Then need to update docopt help message too...

File: README.md [hunk 6]
```diff
@@ -2,135 +2,137 @@
...
+Wow, it was crazy. Now, to update some vendored dependencies, for example,
+`docopt-go` package, manul should be invoked with flag `-U` and import path
+(`github.com/docopt/docopt-go`):
 
 ![update docopt](https://cloud.githubusercontent.com/assets/8445924/10285723/a8ce9f18-6b79-11e5-87ef-2caca393328c.gif)
 
-**manul** can remove specified submodules of vendored dependencies using flag
-`-R` (remove) and specifying dependencies import path.
+**manul** can be used to remove specified submodules of vendored dependencies
+by using flag `-R` (remove) and specifying dependencies import path:
```
Comment (kovetskiy) [inline README.md]: by using `-R` (remove) flag

File: README.md [hunk 7]
```diff
@@ -2,135 +2,137 @@
...
-- `-I [<dependency>...]` - install git submodules for specified or all dependencies
-- `-U [<dependency>...]` - update specified or all already-vendored dependencies.
-- `-R [<dependency>...]` - remove git submodules of specified or all
-    dependencies.
-- `-Q [<dependency>...]` - list all used dependencies.
-- `-C` - detect and remove all git submodules of unused vendored dependencies.
+- `-I [<dependency>...]` - install git submodules for specified/all dependencies;
+- `-U [<dependency>...]` - update specified/all already vendored dependencies;
+- `-R [<dependency>...]` - remove git submodules for specified/all dependencies.
+- `-Q [<dependency>...]` - list all imported dependencies.
```
Comment (kovetskiy) [inline README.md]: what is it mean `imported dependencies`? I think `used dependencies` it's more obvious

File: README.md [hunk 8]
```diff
@@ -2,135 +2,137 @@
...
+- `-C` - detect and remove all git submodules for unused vendored dependencies.
 
 You can see similar help message by passing `-h` or `--help` flag.
 
-## What is it mean «almost go-gettable»
+## What does “almost go-gettable” mean?
```
Comment (kovetskiy) [inline README.md:116]: `?` symbol here should be removed too

File: README.md [hunk 9]
```diff
@@ -2,135 +2,137 @@
...
-golang/go](https://github.com/golang/go/issues/12612) is not solved, you will
-not be able to use `go get` with your project.
+This means, that until [that issue in
+golang/go](https://github.com/golang/go/issues/12612) is not solved yet, you
+will not be able to use `go get` with your project.
 
-BTW, I think it's not trouble, because end-user of your project should not
-install `manul` for compilation from source code.
+But, it's not the big deal, because manul is only required for embedding
+dependencies, not for building or compiling the program. End user of your
```
Comment (kovetskiy) [inline README.md]: `building` is equal to `compiling`, I guess you mean `building or running`

File: README.md [hunk 10]
```diff
@@ -2,135 +2,137 @@
...
+project should not install `manul` for compilation from source code.
 
-Now projects, which uses the git submodules for vendoring dependencies, can be
-builded by running this commands:
+For now projects, which uses the git submodules for vendoring dependencies, can
+be builded by running this commands:
```
Comment (kovetskiy) [inline README.md]: can you add some note like `pretty easy commands`?

**Comment by kovetskiy** [comment]
Thanks, loogs very good, can you also patch `docopt` help message?

File: README.md (hunk 1 above)
Comment (seletskiy) [inline README.md:5]: In my opinion, calling your product `madness` is not a good choice. However, if you insist, I will revert edit.

Comment (kovetskiy) [inline README.md:5]: it's reference to manul (very evil cat), but okay, should not revert.

File: README.md (hunk 2 above)
Comment (seletskiy) [inline README.md]: What the reason for it? I guess, you're confusing this case with ending titles with period, which should not be done.

File: README.md (hunk 4 above)
Comment (seletskiy) [inline README.md]: Agreed.

File: README.md (hunk 3 above)
Comment (seletskiy) [inline README.md]: The main problem is using local search tools, not GitHub, don't you think so?

File: README.md (hunk 9 above)
Comment (seletskiy) [inline README.md]: Oops, yep.

File: README.md (hunk 3 above)
Comment (kovetskiy) [inline README.md]: manul will have same problem with grep utils too

File: README.md (hunk 5 above)
Comment (seletskiy) [inline README.md:86]: Because "already" is an adverb and "already-vendored" is not a compound word.

File: README.md (hunk 6 above)
Comment (seletskiy) [inline README.md]: +

File: README.md (hunk 7 above)
Comment (seletskiy) [inline README.md]: Hmm, yep. But `installed dependencies` is a kidna weird phrasing

File: README.md (hunk 8 above)
Comment (kovetskiy) [inline README.md:116]: ignor this

File: README.md (hunk 7 above)
Comment (kovetskiy) [inline README.md]: I don't know what is it mean `kidna`, but what about `used dependencies`?

Comment (seletskiy) [inline README.md]: It was a typo. I mean "kinda".

Comment (seletskiy) [inline README.md]: Yes, I will ensure, that "used dependencies" is used across all help.

Comment (kovetskiy) [inline README.md]: What about `list all dependencies`?

**Comment by seletskiy** [comment]
docopt is fixed too.

**Comment by kovetskiy** [comment]
Thanks for the contribution!

**Pull request merged**